modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/util/ObjectEntryTestUtil.java

```diff
@@ -31,4 +33,15 @@ public static ObjectEntry addObjectEntry(
 			null, values, ServiceContextTestUtil.getServiceContext());
 	}
 
+	public static ObjectEntry addObjectEntry(
```

Comment:
comment :

it's necesary create this method. Can be called on the testUserWithViewSharingEntryActionCannotViewPrivateModel as it is..